state-chain/pallets/cf-ingress-egress/src/benchmarking.rs

```diff
@@ -462,6 +478,23 @@ mod benchmarks {
 		assert_eq!(PrewitnessedDeposits::<T, I>::iter().count(), 0);
 	}
 
+	#[benchmark]
+	fn create_boost_pools(n: Linear<1, 255>) {
```

Comment:
If we know that the relationship is trivially linear, i.e. `total_weight = weight * count`, we can just benchmark creation of 1 pool and multiply by `process_deposits.len()` (i.e. how it is done in `process_deposit`). By using `Linear` as paramter here, you are asking substrate to run this a large number of times (which is a bit of a waste) to approximate the linear relationship that we already know.